specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.ResourceNotifications/AKSResources.tsp

```diff
@@ -0,0 +1,54 @@
+﻿import "@typespec/versioning";
+import "./AKSResourcesExample.tsp";
+import "./common.tsp";
+using TypeSpec.Versioning;
+
+/**
+Describes the schema of the AKS Fleet Resources events published to Azure Event Grid. This corresponds to the Data property of an EventGridEvent.
+*/
+namespace Microsoft.EventGrid.SystemEvents;
+
+/**
+Schema of the Data property of an EventGridEvent for a Microsoft.ResourceNotifications.AKSResources.FleetGateCreated event.
+*/
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "Maintain compatibility with existing models."
+@example(MicrosoftResourceNotificationsAKSResourcesFleetGateCreatedExample)
+model ResourceNotificationsAksResourcesFleetGateCreatedEventData
```

Comment:
In Go we upper-case acronyms.  So, `Aks` should be `AKS`.

Can you please add entries to https://github.com/Supriya-R-Dixit/azure-rest-api-specs/blob/main/specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/propertyNameOverrideGo.tsp that fixes the casing?  Comment applies to all models that include `Aks`.